src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -192,7 +192,7 @@ private void setBindingsDriver() {
         .whileTrue(new SequentialCommandGroup(new PrintCommand("Running Intake"),
             new IntakeNEO(intakeShooter)));
     new JoystickButton(driverController, Driver.rotateFieldRelative0Deg)
-        .onTrue(new RotateToFieldRelativeAngle(Rotation2d.fromDegrees(0), drivetrain));
+        .onTrue(new AlignToNote(drivetrain));
```

Comment:
You should probably not have the button called "Driver.rotateFieldRelative0Deg" align to note